src/components/ConfirmDialog.js

```diff
@@ -0,0 +1,67 @@
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Typography,
+} from '@mui/material';
+import { useState } from 'react';
+
+const ConfirmDialog = () => {
+  const [mustFilled, setMustFilled] = useState(false);
+
+  const handleClickNo = () => {
+    setMustFilled(false);
+  };
+
+  const handleClickYes = () => {
+    setMustFilled(true);
+  };
+
+  return (
+    <Card
+      sx={{
+        width: 351,
+        height: 175,
+        backgroundColor: '#FFB5B5',
+        borderRadius: '16px',
+      }}
+    >
+      <CardContent>
+        <Typography variant="mediumB" style={{ display: 'block' }}>
```

Comment:
```suggestion
        <Typography variant="mediumB" component='div'>
```
이렇게 하면  스타일은 mediumB, html 태그는 div로 되어서 display: 'block'을 사용하지 않을수 있을 것 같은데 낸시 생각은 어떠신가요??

Reply:
헐 도르 제가 너무 찾던!!! 알고싶었던거엿어요! 저렇게 태그를 지정해줄수있군요!!
도르 말대로 하면 좋을거같아요 시멘틱하게 `p`태그 어떤가요?

Reply:
이런 방법이 있었다니..
`p` 태그 이용해서 수정하였습니다!!
감사해요 도르, 코비 👍